File: backend/LibraryAPI/Models/Book.cs
namespace LibraryAPI.Models
{
    public class Book
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Author { get; set; }

        //foreign key of user
        public int UserId { get; set; }
        public User User { get; set; }

    }
}

File: frontend/src/types/Book.ts
export interface Book {
  id?: number;
  title: string;
  author: string;
  description: string;
  userId?: number;
}

export interface BookFormData {
  title: string;
  author: string;
  description: string;
}

export interface BookUpdateData extends BookFormData {
  id: number;
}
